pkg/setting/setting_k8s_dashboard_cleanup.go

```diff
@@ -0,0 +1,53 @@
+package setting
+
+import (
+	"time"
+)
+
+type K8sDashboardCleanupSettings struct {
+	Interval  time.Duration
+	Timeout   time.Duration
+	BatchSize int64
+}
+
+const (
+	defaultK8sDashboardCleanupInterval  = 30 * time.Second
+	defaultK8sDashboardCleanupBatchSize = int64(10)
+	minK8sDashboardCleanupInterval      = 10 * time.Second
+	minK8sDashboardCleanupTimeout       = 5 * time.Second
+	minK8sDashboardCleanupBatchSize     = int64(5)
+	maxK8sDashboardCleanupBatchSize     = int64(200)
+)
+
+func (cfg *Cfg) readK8sDashboardCleanupSettings() {
+	section := cfg.Raw.Section("k8s_dashboard_cleanup")
```

Comment:
nit - could we rename this to something more generic since it'll be the default in g12?
```suggestion
	section := cfg.Raw.Section("dashboard_cleanup")
```

Reply:
Done